MobiFlight/Joysticks/JoystickOutput.cs

```diff
@@ -9,13 +9,13 @@ namespace MobiFlight
     public class JoystickOutput
     {
         /// <summary>
-        ///  Display name for the output.
+        /// Unique Id for the output.
         /// </summary>
-        public string Label;
+        public string Id;
```

Comment:
Wait this Id is a string?

Reply:
Yes, because our published definition files use strings for output IDs:

```C#
      "Id": "Lights.ParkingBrake",
```

The only thing changing here is the name of the property in the class, from `Name` to `Id` and adjusting the schema. There was a mismatch between what we published in our json files (`Id`) and what the schema was expecting (`Name`).